config/master.d/50-master.conf

```diff
@@ -21,6 +21,7 @@ file_roots:
 pillar_roots:
   base:
     - /usr/share/salt/kubernetes/pillar
+    - /srv/pillar
```

Comment:
If this override needs to be persistent, we'd need to mount this folder from somewhere on the host, why don't use the regular pillar override? We could provide a group of rake tasks that make it easy to configure feature gates, and they'd be saved on the database along with the rest of the pillars.

Note well: I'd love to kill the HTTP pillar API, but that's a different discussion :)